elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base),
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
```

Comment:
I wonder if it would make sense to surface a generically named `accent` property instead of (or in addition to?) the specifically named `BorderBottomColor--hover`.

Reply:
Bump